samples/nrf9160/nrf_cloud_mqtt_multi_service/src/temperature.c

```diff
@@ -0,0 +1,46 @@
+#include <zephyr.h>
+#include <logging/log.h>
+#include <devicetree.h>
+#include <drivers/sensor.h>
+#include <random/rand32.h>
+
+#include "temperature.h"
+
+LOG_MODULE_REGISTER(temperature, CONFIG_MQTT_MULTI_SERVICE_LOG_LEVEL);
+
+#if CONFIG_TEMP_DATA_USE_SENSOR
+
+static const struct device *temp_sensor = DEVICE_DT_GET(DT_ALIAS(temp_sensor));
```

Comment:
blank line required after variable definition

Reply:
Interesting, compliance did not actually complain about this one, but I agree, there should be a newline